src/common/exif.cc

```diff
@@ -1110,6 +1110,19 @@ int dt_exif_write_blob(uint8_t *blob, uint32_t size, const char *path, const int
       dt_remove_exif_keys(imgExifData, keys, n_keys);
     }
 
+    // remove the profile tables
```

Comment:
That's the thing, if you unconditionally remove those tags here, they can never be used by anything.
This is almost alway a good thing BUT.
Our HDR creator writes DNG.
And those dng's are basically only exposure-joined, still in camera color space, not demosaiced.
So theoretically one could use our HDR creator to create HDR DNG from several DNG's (which had those \/ tags), and then process that new DNG in some other software.

So i think these tags should be dropped not when reading, but when writing not into DNG.